tools/stringify.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python
+from __future__ import print_function
+import sys
+import os
+import subprocess
+import platform
+
+dir = './'
+ekey = ''
+defines = {}
+replaced = []
+
+def encrypt(message, key):
+    file = open('./tmp.txt',mode='w')
+    file.write(message)
+    file.close()
+    binary = ''
+    if platform.system() == 'Windows':
+        binary = './adl/contrib/src/easy-encryption/bin/win/ee64'
+    elif platform.system() == 'Darwin':
+        binary = './adl/contrib/src/easy-encryption/bin/macos/ee64'
```

Comment:
I would also suggest to define these paths near the top of this file to make it easier to change and maintain in the future.

Reply:
removed